rpc/v8/block.go

```diff
@@ -91,6 +91,22 @@ type BlockWithReceipts struct {
 	Transactions []TransactionWithReceipt `json:"transactions"`
 }
 
+type BlockWithTxHashesAndReceipts struct {
+	Status rpcv6.BlockStatus `json:"status,omitempty"`
+	BlockHeader
+	TxnHashes    []*felt.Felt             `json:"transactionsHashes"`
+	Transactions []TransactionWithReceipt `json:"transactions"`
+}
+
+func EmptyBlockWithTxHashesAndReceipts() BlockWithTxHashesAndReceipts {
+	return BlockWithTxHashesAndReceipts{
+		Status:       rpcv6.BlockPending,
+		BlockHeader:  BlockHeader{},
+		TxnHashes:    []*felt.Felt{},
```

Comment:
It looks good, why is this an array of `*felt.Felt` instead of `felt.Felt`. Which would be the best use of it

Reply:
Just was looking at implementation of `BlockWithTxHashes` :D , will fix